### PR TITLE
Team

### PR DESCRIPTION
**Pull request by radishevskyi**
# 📝 Description

- added section team

# 🔄 Type of change

- [ ] Cleaning up
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality to something)
- [x] Configuration change
- [x] Translation change

# 🧪 How has this been tested / or will it be tested?

- [ ] Manually surfing the app

# ✅ Checklist:

- [x] My changes generate no new warnings in console or terminal
- [x] I have performed a self-review of my code
- [x] I have updated versions of the package.json and package-lock.json

# 🖼️ Result screenshot/video (optional): 
image

# Gif to summarize my emotions (optional):

### CODE REVIEW

File: locales/pl/team.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (p-osypov) [inline locales/pl/team.json:1]: Translations missing

File: locales/en/team.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (p-osypov) [inline locales/en/team.json:1]: Translations missing

File: src/modules/home/team/team.styles.ts [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+  TitleBlock: styled.div`
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: var(--spacing-x2);
+  `,
+  Title: styled.span`
+    font-size: 48px;
+    background-image: linear-gradient(
```
Comment (p-osypov) [inline src/modules/home/team/team.styles.ts:37]: If this pattern is repeated few times it should be moved to variables.

File: src/modules/home/team/team.styles.ts [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+    width: 100%;
+    margin-bottom: 15px;
+  `,
+  Name: styled.span`
+    font-size: var(--font-size-medium);
+    ${mediaWidth('max', 'xxs')} {
+      font-size: var(--font-size-normal);
+    }
+  `,
+  Position: styled.span`
```
Comment (p-osypov) [inline src/modules/home/team/team.styles.ts:80]: Better rename to 
```suggestion
MilitaryRank: styled.span`
```

File: src/modules/home/team/team.styles.ts [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+    opacity: 70%;
+    font-size: var(--font-size-normal);
+    ${mediaWidth('max', 'xxs')} {
+      font-size: var(--font-size-small);
+    }
+  `,
+
+  JoinUsBtn: styled.button`
+    background-color: var(--color-accent);
+    padding: 20px 40px;
```
Comment (p-osypov) [inline src/modules/home/team/team.styles.ts:90]: Use spacing variable for paddings

File: src/modules/home/team/team.styles.ts [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+    border-radius: 100px;
+    cursor: pointer;
+    font-size: var(--font-size-medium);
+    color: var(--color-white);
+    margin-top: var(--spacing-x4);
+    ${mediaWidth('max', 'xs')} {
+      padding: 17px 30px;
```
Comment (p-osypov) [inline src/modules/home/team/team.styles.ts:97]: Use spacing variable for paddings

File: src/modules/home/team/team.styles.ts [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+    }
+    ${mediaWidth('max', 'xxs')} {
+      padding: 15px 20px;
```
Comment (p-osypov) [inline src/modules/home/team/team.styles.ts:100]: Use spacing variable for paddings

File: src/modules/home/team/team.tsx [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+      <SC.Content>
+        <SliderControls previous={previous} next={next} />
+        <SC.TitleBlock>
+          <Label>{t('team')}</Label>
+          <SC.Title>{t('title')}</SC.Title>
+        </SC.TitleBlock>
+        <SC.SliderContainer>
+          <SC.Slider sliderRef={sliderRef} {...teamSliderSettings}>
+            {teamArray.map((member) => (
+              <SC.Card key={member.name}>
```
Comment (p-osypov) [inline src/modules/home/team/team.tsx:31]:
```suggestion
<SC.Card key={`slider-card-${member.name}`}>
```

File: src/shared/assets/icons/index.tsx [hunk 9]
```diff
@@ -18,6 +19,7 @@ const Icons = {
   FooterEmail,
   Facebook,
   Instagram,
+  CardArrowRight,
```
Comment (p-osypov) [inline src/shared/assets/icons/index.tsx:22]:
```suggestion
MilitaryArrowRight
```

File: src/shared/components/slider/slider.styles.ts [hunk 10]
```diff
@@ -0,0 +1,73 @@
+import { Container } from '@/shared/assets/styles/layout';
+import styled from 'styled-components';
+import { DirectedArrowProps } from './directed-arrow';
+import Icons from '@/shared/assets/icons';
+import { mediaWidth } from '@/shared/assets/styles/mixins';
+
+export const SC = {
+  SliderControls: styled(Container)`
+    position: absolute;
+    top: 52%;
```
Comment (p-osypov) [inline src/shared/components/slider/slider.styles.ts:10]:
```suggestion
top: 52%; // ToDo update this value to 50% after team photos would be changed to square dimension.
```

**Review by p-osypov — CHANGES_REQUESTED** [review]

File: locales/pl/team.json (hunk 1 above)
Comment (radishevskyi) [reply locales/pl/team.json:1]: [feat: added translation](https://github.com/p-osypov/army/pull/7/commits/9fa7a8ccb3bf15df7e6030e4d4368cd4a2204f9d)

File: locales/en/team.json (hunk 2 above)
Comment (radishevskyi) [reply locales/en/team.json:1]: [feat: added translation](https://github.com/p-osypov/army/pull/7/commits/9fa7a8ccb3bf15df7e6030e4d4368cd4a2204f9d)

File: src/shared/components/slider/slider.styles.ts (hunk 10 above)
Comment (radishevskyi) [reply src/shared/components/slider/slider.styles.ts]: [fix: styles](https://github.com/p-osypov/army/pull/7/commits/3da15656f250f7d389212843752c1de57b6decdf)

File: src/shared/assets/icons/index.tsx (hunk 9 above)
Comment (radishevskyi) [reply src/shared/assets/icons/index.tsx]: [fix: logic](https://github.com/p-osypov/army/pull/7/commits/bdaefc0ab0a3454db458111f6b39bbddee8567be)

File: src/modules/home/team/team.tsx (hunk 8 above)
Comment (radishevskyi) [reply src/modules/home/team/team.tsx]: [fix: logic](https://github.com/p-osypov/army/pull/7/commits/bdaefc0ab0a3454db458111f6b39bbddee8567be)

File: src/modules/home/team/team.styles.ts (hunk 7 above)
Comment (radishevskyi) [reply src/modules/home/team/team.styles.ts]: [fix: styles](https://github.com/p-osypov/army/pull/7/commits/3da15656f250f7d389212843752c1de57b6decdf)

File: src/modules/home/team/team.styles.ts (hunk 6 above)
Comment (radishevskyi) [reply src/modules/home/team/team.styles.ts]: [fix: styles](https://github.com/p-osypov/army/pull/7/commits/3da15656f250f7d389212843752c1de57b6decdf)

File: src/modules/home/team/team.styles.ts (hunk 5 above)
Comment (radishevskyi) [reply src/modules/home/team/team.styles.ts]: [fix: styles](https://github.com/p-osypov/army/pull/7/commits/3da15656f250f7d389212843752c1de57b6decdf)

File: src/modules/home/team/team.styles.ts (hunk 4 above)
Comment (radishevskyi) [reply src/modules/home/team/team.styles.ts]: [fix: styles](https://github.com/p-osypov/army/pull/7/commits/3da15656f250f7d389212843752c1de57b6decdf)

File: src/modules/home/team/team.styles.ts (hunk 3 above)
Comment (radishevskyi) [reply src/modules/home/team/team.styles.ts]: [fix: styles](https://github.com/p-osypov/army/pull/7/commits/3da15656f250f7d389212843752c1de57b6decdf)

File: src/modules/home/about-us/about-us.tsx [hunk 11]
```diff
@@ -12,8 +12,8 @@ function AboutUs() {
           <SC.InfoBlock>
             <SC.Logo src="/img/logo2.png" />
             <SC.InfoBlockItem>
-              <SC.Goals>220 цілей</SC.Goals>
-              <SC.Destroyed>знищено</SC.Destroyed>
+              <SC.Goals>{t('goals')}</SC.Goals>
```
Comment (p-osypov) [inline src/modules/home/about-us/about-us.tsx:15]:
```suggestion
<SC.Goals>220 {t('goals')}</SC.Goals>
```

File: src/modules/home/about-us/about-us.styles.ts [hunk 12]
```diff
@@ -101,12 +119,8 @@ export const SC = {
...
 
   Title: styled.h1`
-    font-size: 48px;
-    background-image: linear-gradient(
-      90deg,
-      rgba(255, 255, 255, 1),
-      rgba(255, 255, 255, 0.2)
-    );
+    font-size: var(--font-size-title);
+    background-image: var(--background-gradient);
```
Comment (p-osypov) [inline src/modules/home/about-us/about-us.styles.ts:123]:
```suggestion
background-image: var(--text-background-gradient);
```

**Review by p-osypov — CHANGES_REQUESTED** [review]

File: src/modules/home/about-us/about-us.styles.ts (hunk 12 above)
Comment (radishevskyi) [reply src/modules/home/about-us/about-us.styles.ts]: [fix: styles](https://github.com/p-osypov/army/pull/7/commits/0f3649212e016c3ccbca222f3bc267270fa3e592)

File: src/modules/home/about-us/about-us.tsx (hunk 11 above)
Comment (radishevskyi) [reply src/modules/home/about-us/about-us.tsx]: [fix: logic](https://github.com/p-osypov/army/pull/7/commits/e16b1a2586cc0d0de3cd8af9b9e4e9e3201d80de)

**Review by p-osypov — APPROVED** [review]

**Pull request merged**